README.md

```diff
@@ -115,14 +115,30 @@ Behat tests can be executed from the repository root by running:
 $ ./bin/behat -c tests/behat.yml
 ```
 
-With a single Phing task, you can run every tests suites:
```

Comment:
I put a comment in the ticket. Could you see it and see if you agree on the approach?
If you do not agree, we can see how to address it because it is long journey to fix it.